tests/spread/general/big/rockcraft.yaml

```diff
@@ -9,6 +9,15 @@ license: Apache-2.0
 base: ubuntu:22.04
 platforms:
   amd64:
+
+services:
+  test:
+    override: replace
+    command: sh -c "pwd; sleep 5"
```

Comment:
can you update `task.yaml` to check for this?

Reply:
oops...that was an oversight. thanks. fixed